### PR TITLE
Addresses #2885 NTR liver right lobe sections

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses #2885 NTR liver right lobe sections.

### CODE REVIEW

**Comment by aleixpuigb** [comment]
I have decided to use posterior/anterior nomenclature (in the left lung is lateral and medial) as it seems the standard nomenclature after a cursory review.
image PMID: [30992874](https://pubmed.ncbi.nlm.nih.gov/30992874)

Lateral and medial nomenclature was added as exact synonyms, based on this tesxt in the reference:

image

Of note, some child classes of the left and right liver lobe need revision in a separate ticket.

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -224175,6 +224175,25 @@ relationship: has_part CL:0000067 ! ciliated epithelial cell
...
+synonym: "liver right lateral lobe" EXACT [PMID:24453062]
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+relationship: dc-contributor https://orcid.org/0000-0002-6443-9376
+relationship: part_of UBERON:0001114 ! right lobe of liver
+property_value: dcterms-date "2023-05-15T10:46:34Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600016
+name: liver right anterior lobe
+def: "The anterior sector of the right lobe of the liver." [PMID:30992874]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224191]: Regarding text def for both: sector --> segment

Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:224191]: @aleixpuigb, thank you for clarifying. 

The labels and synonyms include the word "lobe", but the text definition of these classes are describing sectors, not lobes. A similar comment related to brain regions: https://github.com/obophenotype/uberon/issues/2832#issuecomment-1520538497

If the class is meant to represent the sectors, consider renaming the label and synonyms.

Also consider adding synonyms that substitute "liver" with "hepatic" as included in the dbxrefs, e.g., "right hepatic anterior sector" PMID:[30487943](https://pubmed.ncbi.nlm.nih.gov/30487943)

**Review by bvarner-ebi — CHANGES_REQUESTED** [review]
See inline comments.

File: src/ontology/uberon-edit.obo [hunk 2]
```diff
@@ -224175,6 +224175,29 @@ relationship: has_part CL:0000067 ! ciliated epithelial cell
...
+relationship: dc-contributor https://orcid.org/0000-0001-6677-8489
+relationship: dc-contributor https://orcid.org/0000-0002-6443-9376
+relationship: part_of UBERON:0001114 ! right lobe of liver
+property_value: dcterms-date "2023-05-15T10:46:34Z" xsd:dateTime
+
+[Term]
+id: UBERON:8600016
+name: anterior sector of right lobe of liver
+def: "The anterior sector of the right lobe of the liver." [PMID:30992874]
+synonym: "liver right medial lobe" EXACT [https://orcid.org/0000-0002-6443-9376]
```
Comment (bvarner-ebi) [inline src/ontology/uberon-edit.obo:224194]: Consider making this a related, not exact, synonym. Sector vs lobe.

**Pull request merged**